modules/Movecraft/src/main/java/net/countercraft/movecraft/sign/RemoteSign.java

```diff
@@ -100,4 +100,20 @@ public final void onSignClick(PlayerInteractEvent event) {
         
         event.setCancelled(true);
     }
+    private boolean isEqualSign(Sign test, String target) {
+        return !ChatColor.stripColor(test.getLine(0)).equalsIgnoreCase(HEADER) && ( ChatColor.stripColor(test.getLine(0)).equalsIgnoreCase(target)
+                || ChatColor.stripColor(test.getLine(1)).equalsIgnoreCase(target)
+                || ChatColor.stripColor(test.getLine(2)).equalsIgnoreCase(target)
+                || ChatColor.stripColor(test.getLine(3)).equalsIgnoreCase(target) );
+    }
+    private boolean isForbidden(Sign test) {
+        for (int i = 0; i < 4; i++) {
+            for(String s : Settings.ForbiddenRemoteSigns) {
```

Comment:
Convert `test.getLine(i)` to lowercase and check if `Settings.ForbiddenRemoteSigns` contains it.